supervisor/core.py

```diff
@@ -201,7 +203,7 @@ async def start(self):
 
         # Check if system is healthy
         if not self.supported:
-            _LOGGER.critical("System running in a unsupported environment!")
+            _LOGGER.warning("System running in a unsupported environment!")
```

Comment:
This is the only change I'm not sure about, according to the [ADR[(https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) (see the last line) we should just warn.

Reply:
I can life with that